<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод данных БД</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_top">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ввод данных для подключения к базе данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_host">
       <property name="text">
        <string>Имя хоста</string>
       </property>
       <property name="buddy">
        <cstring>txt_host_name</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_host_name"/>
     </item>
     <item>
      <widget class="QLabel" name="lb_bd">
       <property name="text">
        <string>Имя БД</string>
       </property>
       <property name="buddy">
        <cstring>txt_bd_name</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_bd_name"/>
     </item>
     <item>
      <widget class="QLabel" name="lb_login">
       <property name="text">
        <string>Логин</string>
       </property>
       <property name="buddy">
        <cstring>txt_login</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_login"/>
     </item>
     <item>
      <widget class="QLabel" name="lb_pwd">
       <property name="text">
        <string>Пароль</string>
       </property>
       <property name="buddy">
        <cstring>txt_pwd</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_pwd"/>
     </item>
     <item>
      <widget class="QLabel" name="lb_port">
       <property name="text">
        <string>Порт</string>
       </property>
       <property name="buddy">
        <cstring>sb_port</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_port">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64000</number>
       </property>
       <property name="value">
        <number>1234</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_host_name</tabstop>
  <tabstop>txt_bd_name</tabstop>
  <tabstop>txt_login</tabstop>
  <tabstop>txt_pwd</tabstop>
  <tabstop>sb_port</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
